R/adjust_speed.R

```diff
@@ -0,0 +1,56 @@
+
+#' @title Estimate cumtime based on global speed
+#'
+#' @description Estimate cumtime of NA values of a GPS data based on the average
+#' speed of the valid GPS rows.
+#'
+#' @param gps_data A data.table with GPS-like rows.
```

Comment:
it needs to be replaced with:
`#' @param gps_data A GPS-like data.table created with \code{\link{gtfs2gps}}`